content/docs/1.4.0/advanced-resources/fast-replica-rebuild.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Fast Replica Rebuild
+weight: 4
+---
+
+Longhorn supports fast replica rebuilding based on the checksums of snapshot disk files.
+
+## Introduction
+
+The legacy replica rebuilding process walks through all snapshot disk files. Data blocks are hashed in both the local and remote disks to determine if they need to be sent to the remote side. Thus, it is an IO- and computing-intensive process, especially if the volume is large or contains a large number of snapshot files.
```

Comment:
IIRC, the data blocks of each snapshot will be always transferred from a remote replica, then they will be compared with the data blocks of the local snapshot to decide if they should be saved/overridden locally. If yes, then we need to explain this correctly.

Reply:
> IIRC, the data blocks of each snapshot will be always transferred from a remote replica

Client sends a request to server for requesting the remote block's checksum. The checksums are calculated in the remote and local sides, respectively.

Reply:
Clarified, all good!

Reply:
I rephrased the sentences and make them more clear.